gwf/cli/arg_parsing.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import, print_function
+from ..utils import cache, import_object
+from ..core import PreparedWorkflow
+import argparse
+
+
+class SubCommand(object):
```

Comment:
Python 3 has new-style classes by default, so there's no reason to inherit from `object` explicitly.